packages/environment-check/src/checks/environment.ts

```diff
@@ -98,7 +100,7 @@ async function getToolsExtensions(): Promise<{
     const cloudCli = await getCFCliToolVersion();
 
     let toolsExtensions: ToolsExtensions = {
-        nodeVersion: process.versions.node,
+        nodeVersion: nodeVersion,
```

Comment:
We have the node version in two places of the result structure:
`EnvironmentCheckResult->environment->versions->node` and
`EnvironmentCheckResult->environment->toolsExtensions->nodeVersion`

I would avoid having it in two places and delete it here in interface `ToolsExtensions`. If we need to output the node version in multiple places in the markdown report, I would just take it from `EnvironmentCheckResult->environment->versions->node` multiple times to add to the report, but not have it twice in JSON results. That avoids also to pass parameter `nodeVersion` into this function just to write it 1:1 into the result structure.

Reply:
Updated, markdown.ts is not quite as dynamic now if we want to leave the Node.js property in the results table